wca-guidelines.md

```diff
@@ -165,6 +165,8 @@ To be more informative, each Guideline is classified using one of the following
 - A1a4+) [REMINDER] If a competitor has accidentally been permitted to exceed the time limit, the time limit must be enforced retroactively, and the judge, competitor, and WCA Delegate should be informed (see [Regulation 1g2](regulations:regulation:1g2)). Judges must always be aware of the time limit for a current attempt (which might depend on previous attempts, in the case of a cumulative time limit).
 - A1c+) [CLARIFICATION] WCA Delegates should only use their discretion to prevent competitors from being a severe detriment to the competition (e.g. wasting time and/or competition resources). Competitors should not be disqualified for a "poor" result when they are competing to the best of their abilities.
 - A2c1+) [REMINDER] In the past, score sheets have been used to cover puzzles while leaving some sides exposed. This is no longer permitted.
+- A2d1+) [ADDITION] If a missing scramble signature is discovered before an attempt, the puzzle should immediately be returned to a scrambler for verification and a corresponding signature.
+- A2d1++) [ADDITION] If a missing scramble signature is discovered after the beginning of the attempt, the result may be kept, at the discretion of the Delegate. However, the WCA Delegate should not allow such incidents to become common.
```

Comment:
We could suggest looking at video footage. But I don't think it's a good idea to require that (even if available) and the incentives are the wrong way around for it. So I think it's better to leave out.

Reply:
Thanks!